shop/src/main/java/ro/msg/learning/shop/exceptions/ProductsCantBeShipped.java

```diff
@@ -0,0 +1,7 @@
+package ro.msg.learning.shop.exceptions;
+
+public class ProductsCantBeShipped extends RuntimeException{
```

Comment:
Naming of exceptions generally end in ... Exception 😄 . A renaming would be nice for clean code purposes